Tests/UnitTests/Networking/Responses/CustomerInfoDecodingTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfoDecodingTests.swift
+//
+//  Created by Nacho Soto on 4/12/22.
+
+import Nimble
+@testable import RevenueCat
+import SnapshotTesting
+import XCTest
+
+class CustomerInfoDecodingTests: BaseHTTPResponseTest {
+
+    private static let dateFormatter = ISO8601DateFormatter()
+
+    private var customerInfo: CustomerInfo!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+
+        self.customerInfo = try self.decodeFixture("CustomerInfo")
+    }
+
+    func testSchemaVersion() {
+        expect(self.customerInfo.schemaVersion) == "4"
+    }
+
+    func testResponseDataIsCorrect() throws {
```

Comment:
maybe we can spilt this up into multiple cases? the response is likely to grow over time, and the test case is already pretty big